package.json

```
@@ -33,10 +33,10 @@
     "format": "prettier --write \"./**/*.{ts,tsx,js,jsx,json,md,css}\"",
     "generate-contributors": "yarn ts-node --transpile-only ./tools/generate-contributors.ts && yarn all-contributors generate",
     "generate-sponsors": "yarn ts-node --transpile-only ./tools/generate-sponsors.ts",
-    "lint-fix": "eslint . --ext .js,.ts --fix",
+    "lint-fix": "eslint . --ext .js,.jsx,.ts,.tsx --fix",
     "lint-markdown-fix": "yarn lint-markdown --fix",
     "lint-markdown": "markdownlint \"**/*.md\" --config=.markdownlint.json --ignore-path=.markdownlintignore",
-    "lint": "eslint . --ext .js,.ts",
+    "lint": "cross-env NODE_OPTIONS=\"--max-old-space-size=16384\" eslint . --ext .js,.jsx,.ts,.tsx",
```

what do you think about adding this env ci and not using cross-env here?

```suggestion
    "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
```

https://github.com/typescript-eslint/typescript-eslint/blob/03ff45a2e6cc813611378a133d975ea347b86e88/.github/workflows/ci.yml#L181-L182

```yaml
      - name: Lint code
        run: yarn lint
        env:
          NODE_OPTIONS: "--max-old-space-size=16384"
```

---

Scripts in `package.json` are run on developers' devices as well. Their node would also likely OOM, so we can't just fix it for CI